dev-lang/rust-bin/rust-bin-1.25.0.ebuild

```diff
@@ -35,6 +40,15 @@ src_unpack() {
 
 	local postfix
 	use amd64 && postfix=x86_64-unknown-linux-gnu
+	if use arm; then
+		if [[ "$(tc-is-softfloat)" != "no" ]] && [[ ${CHOST} == armv6* ]]; then
```

Comment:
Is there some reason not to branch these if-branches? Right now the formatting is off (with the inner block not indented far enough). I have these already cherry-picked on my local branch, just let me know and we can move forward.

Reply:
Sure, if you can come up with a more reasonable approach that would be a good thing. 

P.S. I forgot to include the toolchain-funcs eclass, in case you're going to cherry-pick.

Reply:
I mean, is there a reason not to just do

```
if use arm && [[ "$(tc-is-softfloat)" != "no" ]] && ${CHOST} == armv6* ]]; then
```

?

Reply:
Indeed, this was my first approach, but something didn't quiet work out. As long as it's only for the bin package, I can give it another try and report back in a few hours.